src/App.js

```diff
@@ -191,25 +190,21 @@ class App extends Component {
         <section id="main" className="container">
           {this.state.articles
             .filter(article => {
-              if (this.state.selectedSource === "All") {
-                return article;
-              } else if (article.source === this.state.selectedSource) {
-                return article;
-              }
-              return null;
+              return this.state.selectedSource === "All"
+                || article.source === this.state.selectedSource
             })
             .filter(article => {
               return article.title
                 .toLowerCase()
-                .includes(this.state.searchText);
+                .includes(this.state.searchText.toLowerCase());
```

Comment:
Lowercasing both searchText and article.title means that your matching is completely case-insensitive. Without this change, if I search with any capital letters nothing matches!